test/core-flags/ignore-warnings/ignore-warnings-flag.test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { run } = require('../../utils/test-utils');
+
+describe('ignore-warnings', () => {
+    it('should ignore the warning emitted', () => {
+        const { exitCode, stderr, stdout } = run(__dirname, ['--ignore-warnings', /Generated Warning/]);
+
+        expect(exitCode).toBe(0);
+        expect(stderr).toBeFalsy();
+        expect(stdout).not.toContain('Module Warning (from ./my-warning-loader.js):');
+        expect(stdout).not.toContain('Generated Warning');
+    });
+
+    it('should reset options.ignoreWarnings', () => {
+        const { exitCode, stderr, stdout } = run(__dirname, ['--ignore-warnings', /Generated Warning/, '--ignore-warnings-reset']);
```

Comment:
`--ignore-warnings-reset`? I think it should be `-no-ignore-warnings`

Reply:
`--ignore-warnings` is not boolean. It can accept an araay. So core provides `--ignore-warnings-reset` just like `--externals-reset` or other reset flags